x/staking/types/exported.go

```diff
@@ -24,7 +24,7 @@ type ValidatorI interface {
 	IsBonded() bool                                                 // check if has a bonded status
 	IsUnbonded() bool                                               // check if has status unbonded
 	IsUnbonding() bool                                              // check if has status unbonding
-	GetOperator() sdk.ValAddress                                    // operator address to receive/return validators coins
+	GetOperator() string                                            // operator address to receive/return validators coins
 	ConsPubKey() (cryptotypes.PubKey, error)                        // validation consensus pubkey (cryptotypes.PubKey)
 	TmConsPublicKey() (cmtprotocrypto.PublicKey, error)             // validation consensus pubkey (CometBFT)
 	GetConsAddr() (sdk.ConsAddress, error)                          // validation consensus address
```

Comment:
Shouldn't the same be done for consAddress? The same hack could be done in the validator struct (I know this PR is about valBech32)

Reply:
i can modify this to be bytes instead since we already have it

Reply:
i didnt do this since its not needed for now. we dont call.String on the method but users can still call it for now. This would cause more breaking changes